src/common/legacy/src/convert_function_to_cnn_network.cpp

```diff
@@ -413,1308 +458,1543 @@ void CNNLayerCreator::on_adapter(const std::string& name,
+                               res->params["pool-method"] = "avg";
+                           }
+                           return res;
+                       });
+    addSpecificCreator({"Select"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+                           LayerParams attrs = {node->get_friendly_name(),
+                                                node->description(),
+                                                details::convertPrecision(node->get_output_element_type(0))};
+                           auto res = std::make_shared<SelectLayer>(attrs);
+                           res->params = params;
+                           return res;
+                       });
+    addSpecificCreator({"BinaryConvolution"},
+                       [](const std::shared_ptr<::ngraph::Node>& node,
+                          const std::map<std::string, std::string>& params) -> CNNLayerPtr {
+                           LayerParams attrs = {node->get_friendly_name(),
+                                                node->description(),
+                                                details::convertPrecision(node->get_output_element_type(0))};
+                           auto res = std::make_shared<BinaryConvolutionLayer>(attrs);
+
+                           // todo: investigate difference between ngraph parameters for BinConvolution and the
+                           // implementation above this leads to accuracy issue for
+                           // Precollected_ONNX_ResNet50_88percentinto1bit e2e test res->params = params;
+
+                           auto castedLayer = ::ngraph::as_type_ptr<::ngraph::op::v1::BinaryConvolution>(node);
+                           IE_ASSERT(castedLayer)
+                               << " Operation " << node->description() << " with name " << node->get_friendly_name()
+                               << " cannot be casted to ngraph::op::v1::BinaryConvolution";
+
+                           std::string value;
+                           for (const auto& val : castedLayer->get_pads_begin()) {
+                               if (!value.empty())
+                                   value += ",";
+                               value += Builder::asString(val);
+                           }
+                           res->params["pads_begin"] = value;
+
+                           value.clear();
+                           for (const auto& val : castedLayer->get_pads_end()) {
+                               if (!value.empty())
+                                   value += ",";
+                               value += Builder::asString(val);
+                           }
+                           res->params["pads_end"] = value;
+
+                           switch (castedLayer->get_auto_pad()) {
+                           case ::ngraph::op::PadType::SAME_UPPER:
```

Comment:
@popovaan Please revert codestyle changes. hard to understand required changes

Reply:
Done.